assets/js/ui/search.ts

```diff
@@ -0,0 +1,18 @@
+export default function initSearchFields(el: HTMLElement) {
+  el.querySelectorAll<HTMLInputElement>("input[type=search]").forEach(
```

Comment:
It also submits the form in this situation:

* the search input is empty
* you type something
* you clear it by clicking the clear icon

in this case there is no reason to submit anything as the input value hasn't changed

Not sure if you can detect this during a search event

Reply:
@mietcls  Is the type=search and clear button also needed in the add/update contributor form?

Reply:
> @mietcls Is the type=search and clear button also needed in the add/update contributor form?

@nicolasfranck You specifically mean for the first name and last name filter fields here?

Reply:
Ah yes, that would be nice. I can create a separate issue so we don't have to wait to merge this one.